templates/aai-secret.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "sda.fullname" . }}-aai
+type: Opaque
+data:
+  id: NWUzZThhMTFmNzMwNDJlMTYyMDYxZTVhNzA3NGE0ZDVlZWU1NDE2OQ==
+  secret: MzQzYmI1ZjI3ZGEyMTA3NDEwYTM4MGJjOTlmZGQ0ZTg2YmE5YjhkZQ==
```

Comment:
Hohum.

Reply:
for test, reset just in case

Reply:
Oops. Removed. Credentials also reset. Glad they were test ones. Need to parameterize these too